content/docs/languages/go/quickstart.md

```diff
@@ -126,18 +126,34 @@ Remember to save the file!
 Before you can use the new service method, you need to recompile the updated
 `.proto` file.
 
-While still in the `examples/helloworld` directory, run the following command:
+While still in the `examples/helloworld` directory, run the following commands:
 
 ```sh
-$ protoc --go_out=plugins=grpc:. --go_opt=paths=source_relative helloworld/helloworld.proto
+$ ( cd ../../cmd/protoc-gen-go-grpc && go install . )
+$ protoc \
+  --go_out=Mgrpc/service_config/service_config.proto=/internal/proto/grpc_service_config:. \
+  --go-grpc_out=Mgrpc/service_config/service_config.proto=/internal/proto/grpc_service_config,requireUnimplementedServers=false:. \
```

Comment:
Let's remove the `requireUnimplementedServers=false`option here.  I don't want to train people to use this un-recommended flag.  The only reason we use it in `regenerate.sh` is to avoid breaking people not already doing it (and also we may change the way it works!).

Reply:
Done